spec/features/lettings_log_spec.rb

```diff
@@ -110,15 +110,55 @@
     end
 
     context "when completing the setup log section" do
-      it "does not include the organisation and created by questions" do
-        visit("/lettings-logs")
-        click_button("Create a new lettings log")
-        click_link("Set up this lettings log")
-        log_id = page.current_path.scan(/\d/).join
-        expect(page).to have_current_path("/lettings-logs/#{log_id}/needs-type")
-        visit("lettings-logs/#{log_id}/setup/check-answers")
-        expect(page).not_to have_content("Owning organisation #{user.organisation.name}")
-        expect(page).not_to have_content("Log owner #{user.name}")
```

Comment:
Not showing the owning organisation and created by questions does not apply in all cases - it depends on how many potential owning organisations there are